src/core/movieApiService.js

```diff
@@ -0,0 +1,49 @@
+import axios from "axios";
+
+let movieAxiosInstanse = {};
+
+const MovieApiService = {
+  init(options) {
+    movieAxiosInstanse = axios.create({
+      timeout: 5000,
+      ...options,
+      baseURL: "http://react-cdp-api.herokuapp.com"
+    });
+
+    // API doesn't work with single 'filter' query param
```

Comment:
If you want to use arrays in query parameters with Axios, take a look at "paramsSerializer" in the config object ("x" in your case)
https://github.com/axios/axios#request-config

Reply:
The API call with a filter parameter works very strange as for me. I even found [sources](https://github.com/avg206/movie-api-server/blob/master/src/api/middleware/filter.js) and at one moment was close to figure out what exactly is wrong :)
So I decided to replace it with simple request with ```searchBy='genres'```and```search='GENRE_NAME'``` params. It looks like everything works. And forgot to remove this interceptor